clang/test/dpct/launch-kernel-cooperative-usm.cu

```diff
@@ -83,9 +83,13 @@ int main() {
   void *kernel_func = (void *)&kernel;
 
   // CHECK: /*
-  // CHECK-NEXT: DPCT1007:{{[0-9]+}}: Migration of cudaLaunchCooperativeKernel is not supported.
+  // CHECK-NEXT: DPCT1123:{{[0-9]+}}: The kernel function pointer cannot be deduced correctly. You need cast "kernel_func" to a function pointer, and may adjust the kernel function argument(s) and the dimension of the sycl::nd_item type manually.
```

Comment:
The kernel function pointer cannot be deduced correctly. You need cast "kernel_func" to a function pointer. 
According to the kernel function definition, adjusting the kernel function arguments and the dimension of the sycl::nd_item may be also required.